modelscope_agent/llm/base.py

```diff
@@ -34,7 +34,10 @@ class BaseChatModel(ABC):
 
     """
 
-    def __init__(self, model: str, model_server: str):
+    def __init__(self,
+                 model: str,
+                 model_server: str,
+                 support_fn_call: bool = None):
         self._support_fn_call: Optional[bool] = None
```

Comment:
The support_fn_call should be assigned to self._support_fn_call, such as
self._support_fn_call: Optional[bool] = support_fn_call
?

Reply:
fixed